tests/test_web_request.py

```diff
@@ -820,3 +821,47 @@ async def invalid_handler_1(request):
 
     async with client.get("/1") as resp:
         assert 500 == resp.status
+
+
+@pytest.mark.parametrize(
+    "header,header_attr",
```

Comment:
FTR it's usually better readable (though lesser known) to use tuples in parameterize:
```suggestion
    ("header", "header_attr"),
```